lib-rgbpanel/src/rgbpanel.cpp

```diff
@@ -44,7 +46,8 @@ RgbPanel::RgbPanel(uint32_t nColumns, uint32_t nRows, uint32_t nChain, RgbPanelT
 	m_tType(tType),
 	// Text
 	m_nMaxPosition(nColumns / FONT_CP437_CHAR_W),
-	m_nMaxLine(nRows / FONT_CP437_CHAR_H)
+	m_nMaxLine(nRows / FONT_CP437_CHAR_H),
+	m_nFontID(0)
```

Comment:
initialization should be done in .h file

Reply:
Fixed